<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>555</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Задачник</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="title_label">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Задачник</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="add_group">
      <property name="title">
       <string>Добавить задачу</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="task_input">
         <property name="placeholderText">
          <string>Введите новую задачу...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="category_combo"/>
       </item>
       <item>
        <widget class="QPushButton" name="add_btn">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="category_group">
      <property name="title">
       <string>Управление категориями</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="add_category_btn">
         <property name="text">
          <string>+ Категория</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_category_btn">
         <property name="text">
          <string>− Категория</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="filter_group">
      <property name="title">
       <string>Фильтры</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="filter_label">
         <property name="text">
          <string>Фильтр:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="filter_combo">
         <item>
          <property name="text">
           <string>Все задачи</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Активные</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Выполненные</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По категориям</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="category_filter_label">
         <property name="text">
          <string>Категория:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="category_filter">
         <item>
          <property name="text">
           <string>Все категории</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="tasks_list">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="actions_group">
      <property name="title">
       <string>Действия</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="clear_completed_btn">
         <property name="text">
          <string>Очистить выполненные</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refresh_btn">
         <property name="text">
          <string>Обновить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>464</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
